gossipd/routing.c

```diff
@@ -154,6 +154,16 @@ struct chan *next_chan(const struct node *node, struct chan_map_iter *i)
 	return chan_map_next(&node->chans.map, i);
 }
 
+static void destroy_routing_state(struct routing_state *rstate)
+{
+	/* Since we omitted destructors on these, clean up manually */
```

Comment:
Comment could probably be better.  "Since we omitted destructors on the channels, clean them up here when we clean up the routing state".  It seem, inside a destructor itself, so word "manually" feels wrong.